docs/admin/install/troubleshoot.rst

```diff
@@ -1,6 +1,114 @@
 Troubleshooting Installation Errors
 ===================================
 
+Error configuring SecureDrop Workstation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Failed to import Submission Private Key
+---------------------------------------
+
+If importing the submission key  using ``sdw-admin --configure`` fails, you can also copy the submission key manually.
+
+- Open a ``dom0`` terminal by opening the **Q Menu**, selecting the gear icon on the left-hand side, then selecting **Other > Xfce Terminal**. Once the Terminal window opens, run the following command to list the SVS submission key details, including its fingerprint:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault \
+      "gpg --homedir /run/media/user/TailsData/gnupg -K --fingerprint"
+
+- Next, run the comand:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io vault \
+      "gpg --homedir /run/media/user/TailsData/gnupg --export-secret-keys --armor <SVSFingerprint>" \
+      > /tmp/sd-journalist.sec
+
+  where ``<SVSFingerprint>`` is the submission key fingerprint, typed as a single unit without whitespace. This will copy the submission key in ASCII format to a temporary file in dom0, ``/tmp/sd-journalist.sec``.
+
+- Verify the that the file starts with ``-----BEGIN PGP PRIVATE KEY BLOCK-----`` using the command:
+
+  .. code-block:: sh
+
+    head -n 1 /tmp/sd-journalist.sec
+
+- Unmount the SVS USB 
+
+- Run the following command in the ``dom0`` terminal:
+
+  .. code-block:: sh
+
+    sudo cp /tmp/sd-journalist.sec /usr/share/securedrop-workstation-dom0-config/
+
+- Proceed with :ref:`configuring the workstation<manual_configure>`
```

Comment:
If they failed to import the submission key with `sdw-admin --configure`, the journalist interface details also will likely not have been imported yet, as the script [does this](https://github.com/freedomofpress/securedrop-workstation/blob/90ae1a9bde063faec52be8a3ca75d8b0b26bd2bb/files/sdw-admin.py#L463) only after a submission key has been successfully configured.

So I would suggest instead wording this as follows:

> You can run `sdw-admin --configure` to now import the Journalist Interface details. Alternatively, follow the steps below to do so. Once both Submission Key and Journalist Interface details are configured, proceed with configuring the workstation.